bfvmm/include/test/intrinsics.h

```diff
@@ -41,6 +41,40 @@ uint16_t g_tr;
 ::x64::gdt_reg::reg_t g_gdtr{};
 ::x64::idt_reg::reg_t g_idtr{};
 
+extern "C" void _handle_nmi(void) noexcept {}
```

Comment:
no needed

Reply:
@rianquinn this is a stub for unit tests